src/Uno.Extensions.RuntimeTests.Core/GlobalUsings.cs

```diff
@@ -0,0 +1 @@
+﻿global using ListView = Microsoft.UI.Xaml.Controls.ListView;
```

Comment:
Why is it needed ? Sounds like a big hack and we are going to face issue with all controls we are referencing in code. We should probably disable implicit usings for the whole runtime tests app, not only wasm.